deploy/cloudformation/manifest-pipeline.yml

```diff
@@ -1060,13 +1060,65 @@ Resources:
         Variables:
           SSM_KEY_BASE: !Ref AppConfigPath
           SENTRY_DSN: !Ref SentryDsn
-          STATE_MACHINE: !Sub "arn:aws:states:${AWS::Region}:${AWS::AccountId}:stateMachine:${AWS::StackName}-StateMachineCreateCSVs"
+          STATE_MACHINE: !Sub "arn:aws:states:${AWS::Region}:${AWS::AccountId}:stateMachine:${AWS::StackName}-MarbleSourceSystemHarvest"
 
 
-  StateMachineCreateCSVs:
+  CollectionsApiLambdaFunction:
+    Type: AWS::Serverless::Function
+    Properties:
+      Handler: handler.run
+      Runtime: python3.7
+      Timeout: 900
+      CodeUri: collections_api/
+      Policies:
+        - Version: 2012-10-17
+          Statement:
+            - Effect: Allow
+              Action:
+                - 's3:ListObjects'
+                - 's3:ListBucket'
+              Resource:
+                - !Sub "arn:aws:s3:::${RBSCS3ImageBucketName}"
+                - !Sub "arn:aws:s3:::${RBSCS3ImageBucketName}/*"
+        - Version: 2012-10-17
+          Statement:
+            - Effect: Allow
+              Action:
+                - 'states:ListStateMachines'
+                - 'states:StartExecution'
+              Resource: !Sub "arn:aws:states:${AWS::Region}:${AWS::AccountId}:stateMachine:${AWS::StackName}-MarbleSourceSystemHarvest"
+        - Version: 2012-10-17
+          Statement:
+            - Effect: Allow
+              Action:
+                - "ssm:GetParametersByPath"
+              Resource:
+                - !Sub "arn:aws:ssm:${AWS::Region}:${AWS::AccountId}:parameter${AppConfigPath}/*"
+                - !Sub "arn:aws:ssm:${AWS::Region}:${AWS::AccountId}:parameter${MarbleProcessingKeyPath}/*"
+        - Version: 2012-10-17
+          Statement:
+            - Effect: Allow
+              Action:
+                - "s3:PutObject"
+                - 's3:GetObject'
+                - "s3:ListBucket"
+                - "s3:ListBucketVersions"
+              Resource:
+                - !Join ["", [!GetAtt ProcessBucket.Arn, "/*"]]
+                - !Join ["", [!GetAtt ManifestBucket.Arn, "/*"]]
+                - !GetAtt ProcessBucket.Arn
+                - !GetAtt ManifestBucket.Arn
+      Environment:
+        Variables:
+          SSM_KEY_BASE: !Ref AppConfigPath
+          SENTRY_DSN: !Ref SentryDsn
+          STATE_MACHINE: !Sub "arn:aws:states:${AWS::Region}:${AWS::AccountId}:stateMachine:${AWS::StackName}-MarbleSourceSystemHarvest"
```

Comment:
I didn't see in the lambda anywhere this STATE_MACHINE env var was being used. Is this needed?